geometry_msgs/Pose target
float64 gripper_state